src/plugins/intel_gpu/src/kernel_selector/kernels/swiglu/swiglu_kernel_base.h

```diff
@@ -6,18 +6,18 @@
 
 #include "kernel_base_opencl.h"
 #include "kernel_selector_params.h"
-#include "intel_gpu/op/swiglu.hpp"
+#include "ov_ops/swiglu.hpp"
```

Comment:
TO FIX: This header is not visible here [log](https://openvino-ci.toolbox.iotg.sclab.intel.com/blue/organizations/jenkins/private-ci%2Fie%2Fbuild-linux-rhel84/detail/build-linux-rhel84/4196/pipeline/489/#step-494-log-111)
```
intel_gpu/src/kernel_selector/kernels/swiglu/swiglu_kernel_base.h:9:10: 
fatal error: ov_ops/swiglu.hpp: No such file or directory
```

Reply:
TO FIX: This header is not visible here
```
intel_gpu/src/kernel_selector/kernels/swiglu/swiglu_kernel_base.h:9:10: 
fatal error: ov_ops/swiglu.hpp: No such file or directory
```